terraform.md

```diff
@@ -0,0 +1,120 @@
+# Terraform
+
+Terraform is a really powerful tool with a lot of freedom in the way you use it.
+It isn't a very opinionated tool, however if you don't stick with rules it can
+easily get complicated and hard to maintain.
+
+This is why we decided to write our own guideline on how to structure a
+`terraform` codebase.
+
+## File Structure and Filenames
```

Comment:
🐼 House convention for titles is to use sentence-case.

Reply:
Actually, we should use titles that describe the pattern. So this could be "Name files after the resource they manage"